src/utils/addRandomCell.tsx

```diff
@@ -0,0 +1,34 @@
+import type { CellsType } from '../components/types/GameType';
+import getEmptyCellsIndex from './getEmptyCells';
+
+export const addTwoRandomCells = (cells: CellsType): CellsType => {
+  const emptyCells = getEmptyCellsIndex(cells);
+
+  const randomIndices = emptyCells.sort(() => Math.random() - 0.5).slice(0, 2);
+
+  const newCells = [...cells].map((row) => [...row]);
+  randomIndices.forEach(([rowIndex, colIndex]) => {
+    // 이 조건 깔끔하게 처리할 수 있는 방법 확인
+    if (newCells[rowIndex] !== undefined) {
+      newCells[rowIndex][colIndex] = 2;
+    }
+  });
```

Comment:
```suggestion
  const newCells = cells.map((row, i) =>
    row.map((cell, j) =>
      randomIndices.some((indice) => indice[0] === i && indice[1] === j)
        ? 2
        : cell,
    ),
  );
```

`addOneRandomCell` 에도 동일한 의견입니다!